torch/_inductor/codegen/mps.py

```diff
@@ -63,6 +63,11 @@ def _print_ModularIndexing(self, expr: sympy.Expr) -> str:
         mod = self.doprint(mod)
         return f"({x}) % ({mod})"
 
+    def _print_Min(self, expr: sympy.Expr) -> str:
+        if len(expr.args) != 2:
+            raise RuntimeError("Min only supported for 2 or 3 args")
```

Comment:
You mean only for 2 args? [not 2 or 3]

Reply:
Good point, let me update the message (I can add `min3`, but want to see if it's actually used anywhere